DeepCat.js

```diff
@@ -583,6 +583,17 @@
 	}
 
 	/**
+	 * Hash function for generating hint box cookie token.
```

Comment:
fixed this in https://github.com/wmde/DeepCat-Gadget/commit/829cd78d38f8e86fd6efee3d673d549ac8f56297 :->